### ISSUE TITLE
Can't calibrate PA with PETG

### ISSUE THREAD

**Issue by greenbigfrog** [Fragmon/calibration#7]
Hi, thanks for your work.
The PA calibration worked very well for PLA, but with PETG I can't get reliable results out of the print resulting from the calibration script.
I'm printing eSUN PETG, fresh out of the box, at 240/70 on a heavily modded ender 3 with e3d hemera.
I've tried various steps, which I've attached the best photo of that I could take.
I'm also attaching the current script, so that the config values can be read.
[calibrate.txt](https://github.com/Fragmon/calibration/files/10414044/calibrate.txt)

Now that I'm thinking about it, the issue probably is just the too high extruder acceleration, for these temps with my extrusion setup. But perhaps you have some insight for me. Thanks.

### 0.005 steps
![0.005steps](https://user-images.githubusercontent.com/6208894/212383874-ea019842-3483-4ad0-ab5d-884b5cf5a4fe.jpg)
### 0.01 steps
![0.01steps](https://user-images.githubusercontent.com/6208894/212383756-a763fa38-bbeb-4272-8908-c9ecc5543b7c.jpg)
### 0.05 steps
![0.05steps](https://user-images.githubusercontent.com/6208894/212383811-5184bc60-3ec9-489c-ae5e-f53aacbdaddf.jpg)
(Some of the images have the wrong orientation, not sure why...)

**Comment by Fragmon** [comment]
Yes petg dont like this acclereration. Try to reduce the speed between inner and outer speed.
